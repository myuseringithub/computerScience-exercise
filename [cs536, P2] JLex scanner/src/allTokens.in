{
}
(
)
;
,
.
<<
>>
++
--
+
-
*
/
!
&&
||
==
!=
<
>
<=
>=
=
a
z
green
Red
lowercase
camelCase
PascalCase
snake_case_name
mix1_2_3
_
___
_x_
_1
_z
_underscore123
0
1
2
3
4
5
6
7
8
9
1234567890
100
1000
1000000000
int
bool
if
return
true
cin
void
else
cout
while
false
struct
"very long string "
123
""
"&!88"
"use \n to denote a newline character"
"include a quote like this \" and a backslash like this \\"
"/"
"// 1234 ## 123"
"          "
"##nocommenthere"
