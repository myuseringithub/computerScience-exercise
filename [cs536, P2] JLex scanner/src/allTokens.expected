  1: 1	{
  2: 1	}
  3: 1	(
  4: 1	)
  5: 1	;
  6: 1	,
  7: 1	.
  8: 1	<<
  9: 1	>>
 10: 1	++
 11: 1	--
 12: 1	+
 13: 1	-
 14: 1	*
 15: 1	/
 16: 1	!
 17: 1	&&
 18: 1	||
 19: 1	==
 20: 1	!=
 21: 1	<
 22: 1	>
 23: 1	<=
 24: 1	>=
 25: 1	=
 26: 1	a
 27: 1	z
 28: 1	green
 29: 1	Red
 30: 1	lowercase
 31: 1	camelCase
 32: 1	PascalCase
 33: 1	snake_case_name
 34: 1	mix1_2_3
 35: 1	_
 36: 1	___
 37: 1	_x_
 38: 1	_1
 39: 1	_z
 40: 1	_underscore123
 41: 1	0
 42: 1	1
 43: 1	2
 44: 1	3
 45: 1	4
 46: 1	5
 47: 1	6
 48: 1	7
 49: 1	8
 50: 1	9
 51: 1	1234567890
 52: 1	100
 53: 1	1000
 54: 1	1000000000
 55: 1	int
 56: 1	bool
 57: 1	if
 58: 1	return
 59: 1	true
 60: 1	cin
 61: 1	void
 62: 1	else
 63: 1	cout
 64: 1	while
 65: 1	false
 66: 1	struct
 67: 1	"very long string "
 68: 1	123
 69: 1	""
 70: 1	"&!88"
 71: 1	"use \n to denote a newline character"
 72: 1	"include a quote like this \" and a backslash like this \\"
 73: 1	"/"
 74: 1	"// 1234 ## 123"
 75: 1	"          "
 76: 1	"##nocommenthere"
 78: 1	-
 79: 1	variable
 79: 9	"quote"
 79:16	123
 79:19	_variable
 80: 5	space
 80:16	all
 80:26	over
 80:35	12345678
 81: 1	while
 81: 6	(
 81: 7	false
 81:12	)
 81:14	{
 82: 5	int
 82: 9	x
 82:11	=
 82:13	1
 82:14	+
 82:16	5
 82:18	/
 82:20	2
 82:22	*
 82:23	*
 82:25	9399420
 82:32	;
 83: 5	return
 83:12	"are numbers correct ?"
 83:35	;
 84: 1	}
